tests/unittest_brain.py

```diff
@@ -2994,6 +2994,29 @@ def test_infer_dict_from_keys() -> None:
 
 
 class TestFunctoolsPartial:
+    def test_infer_partial(self) -> None:
```

Comment:
This can be a `staticmethod`. `self` isn't used.

Reply:
I got used to `pylint` warning me about this. Forgot it is turned off for `astroid`.

Reply:
Sometimes it's helpful, but not always. I opened an issue in pylint a while back: https://github.com/PyCQA/pylint/issues/5502